src/Controller/InfoController.php

```diff
@@ -43,15 +42,23 @@
 class InfoController extends AbstractController
 {
     private array $systemInfo = [];
+    private readonly string $buildPath;
 
     public function __construct(
-        private readonly string $buildPath,
+        #[Autowire(param: 'kernel.project_dir')]
```

Comment:
It still takes some getting used to. My personal pref still is using binded named values in the services.yaml. But this solution scores low on obscurity.

Reply:
And it is the way of the future. You can still overwrite this in the services.yaml if you need to...

Reply:
I also think it needs some time to get used to it. I felt exactly the same when I first saw attrutes at all.